examples/config_standard_elements.yaml

```diff
@@ -0,0 +1,137 @@
+# This is an example of the template specification file for the standard cm-service processing setup
+# for 'Elements', i.e., Jobs, Group, Steps, Campaign
+
+
+# Import the common script templates
+- import: "${CM_CONFIGS}/config_standard_scripts.yaml"
+# Now define the templates for the standard elements
+- SpecBlock:
+      # Job Template, runs a single workflow
+      name: job
+      handler: lsst.cmservice.handlers.job_handler.JobHandler
+      collections:
+          # This is the output RUN collection for the Job
+          job_run: "{root}/{campaign}/{step}/{group}/{job}"
+      scripts:
+          # The bps script and configuration
+          - Script:
+                name: bps
+                spec_block: panda_script
+                collections:
+                    run: "{job_run}"
+                    inputs: ["{step_input}", "{campaign_input}", "{campaign_ancillary}"]
+          # Uses bps report to collect information on the bps job and know when it is done
+          - Script:
+                name: bps_report
+                spec_block: panda_report_script
+                prerequisites: ['bps']
+                collections:
+                    run: "{job_run}"
+                    inputs: ["{step_input}", "{campaign_input}", "{campaign_ancillary}"]
+          # Uses manifest checker to gather information about the workflow
+          - Script:
+                name: manifest_report
+                spec_block: manifest_report_script
+                prerequisites: ['bps_report']
+                collections:
+                    run: "{job_run}"
+      data:
```

Comment:
How does `rescue: false` also fall under `data` ?

Reply:
We could chose to define a job that has `rescue: true`, which would use the same template, but just override that one parameter,